pkgs/by-name/to/toml-sort/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+let
+  version = "0.23.1";
+in
+python3Packages.buildPythonPackage {
+  pname = "toml-sort";
+  inherit version;
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
Help to avoid legacy build.